pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -1310,11 +1310,11 @@ func (og *operationGenerator) GenerateVerifyControllerAttachedVolumeFunc(
 		if !volumeToMount.ReportedInUse {
 			// If the given volume has not yet been added to the list of
 			// VolumesInUse in the node's volume status, do not proceed, return
-			// error. Caller will log and retry. The node status is updated
-			// periodically by kubelet, so it may take as much as 10 seconds
-			// before this clears.
-			// Issue #28141 to enable on demand status updates.
-			return volumeToMount.GenerateError("Volume has not been added to the list of VolumesInUse in the node's volume status", nil)
+			// nil. Kubelet will retry in the following loops.
+
+			// Issue #75230 exponentialBackOff would lead to
+			// driver implementing attacher not mounting volume in time.
+			return nil, nil
```

Comment:
Because if the func returns not nil, it will trigger exponentialBackOffError. Thus, when `mount` function will be called is largely decided by how log `attach` function will last. If `attach` function lasts 8s, then, the `mount` function(0.5 +1 +2 +4 < 8, retry for 4 times) is not ready and it would be called 8s later. If `attach` function lasts 16s, attach function would be called 16s later. it is Intolerable for users.